webtau-open-api/src/main/java/com/twosigma/webtau/openapi/OpenApiSpecConfig.java

```diff
@@ -26,12 +26,16 @@
 
 public class OpenApiSpecConfig implements WebTauConfigHandler {
     static final ConfigValue specUrl = declare("openApiSpecUrl",
-            "url of OpenAPI 2 spec against which to validate http calls", () -> "");
+            "url of OpenAPI 2 spec against which to validate http calls", () -> null);
```

Comment:
a bit confused. `null` when used in `getAsString` will still return empty string. 
Could you please explain how does it fix the problem?

Reply:
That’s my bad.  I initially had a different fix where I was using `getAsObject` but then changed the approach and forgot to roll this bit back.  I’ll fix